opta/commands/init_templates/variables/azure_tenant_id.py

```diff
@@ -0,0 +1,11 @@
+from opta.commands.init_templates.helpers import dictionary_deep_set
+from opta.commands.init_templates.template import TemplateVariable
+
+
+def apply(d: dict, v: str) -> dict:
+    set_path = dictionary_deep_set(["providers", "azurerm", "tenant_id"])
+    set_path(d, v)
+    return d
+
+
+azureTenantIdVariant = TemplateVariable(prompt="Azure tenant ID", applier=apply,)
```

Comment:
add this link https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-create-new-tenant

Reply:
86ecd7c0244285f0bd5f7fa8fb8b6903cc96a974